BUILDING.md

```diff
@@ -42,32 +42,60 @@ browserify index.js > bundle.js
 
 ## Angular CLI
 
-Currently Angular CLI uses Webpack under the hood to bundle and build your Angular application.
-Sadly it doesn't allow you to override its Webpack config in order to add the plugin mentioned in the [Webpack](#webpack) section.
-Without this plugin your build will fail when it tries to build glslify for WebGL plots.
+Since Angular uses webpack under the hood and doesn't allow easily to change it's webpack configuration. There is some work needed using a custom-webpack builder to get things going.
```

Comment:
```suggestion
Since Angular uses webpack under the hood and doesn't allow easily to change it's webpack configuration, there is some work needed using a `custom-webpack` builder to get things going.
```